### ISSUE TITLE
Add labels to time entries using Outlook categories

### ISSUE THREAD

**Issue by olemp** [Puzzlepart/did#283]
### TLDR ### 
Possible for the user to tag hours from outlook, but in a controlled manner. 

Usecase: Overtime, Non-invoicable hours, etc etc

### Solution ###
With issue #201 we added support for Labels in Did. Currently we can add labels to projects, which can in turn be used to group different projects, e.g. all projects where we have to punch in hours in an external system (Crayon)

We suggest expanding the use of labels to time entries. If a user adds a category which matches a label in Did to an event , that label is applied to that time entry in did.

**Testcase**
1. Create a label in Did 365
2. Add an event tagged with a project which does not have a label
3. Add a category in Outlook which matches the label created
4. Verify in Did 365 that we have applied a category to the time entry

### Proposed presentation in Timesheet
![image](https://user-images.githubusercontent.com/7606007/84029720-238dbe80-a993-11ea-9236-b3e272db60d2.png)
